pkg/controller/nodeipam/ipam/BUILD

```diff
@@ -37,8 +37,8 @@ go_library(
         "cloud_cidr_allocator.go",
         "controller.go",
         "doc.go",
+        "multi_range_allocator.go",
         "range_allocator.go",
-				"multi_range_allocator.go",
```

Comment:
@khenidak this hunk should be done in the commit before where multi_range_allocator.go gets added, not here.